example/eks/main.tf

```diff
@@ -24,7 +24,6 @@ module "fury_example" {
       min_size : 1
       max_size : 2
       instance_type : "m5.large"
-      spot_instance: true
```

Comment:
being that this file is an example, I'd leave this parameter explicit, so people know that they can change it.

Reply:
The second node pool is named `-spot` and has this parameter explicitly.

Reply:
cool, the diff only showed this section and I did not check the whole file, that's even better!